### ISSUE TITLE
Implement NodePieces

### ISSUE THREAD

**Issue by ChrisDelClea** [Accenture/AmpliGraph#267]
**Background and Context*

Hey guys, 

would be awesome if you could also implement the inductive version of NodePieces (https://github.com/migalkin/NodePiece). 

**Description**

Background: i have a task where i want to find the most similar nodes/entities to a given, unseen node/entity in the graph. 

Best regards
Chris

**Comment by lukostaz** [comment]
I agree. One of the most interesting works that came out in the past year. On our radar.
